requirements.in

```diff
@@ -0,0 +1,5 @@
+bump2version==1.0.0
```

Comment:
should go with `bump-my-version` here? since `bump2version` is no longer maintained? I think that is what we are switching on other projects?

Reply:
Got a shot.

`bump-my-version` needs Pydantic v2.0+. So I will need to upgrade polyswarm-artifact to do so.

Going there, then will be back here.